core/src/core-plugin-definitions.ts

```diff
@@ -1031,7 +1031,7 @@ export interface LocalNotification {
   smallIcon?: string;
   attachments?: LocalNotificationAttachment[];
   actionTypeId?: string;
-  extra?: any;
+  extra?: Record<string, string>;
```

Comment:
At the moment it's fetched as a JSObject, so I think it should be 
```suggestion
  extra?: Record<string, any>;
```
I've tested passing object and they are correctly fetched, so I don't think we should limit it to just strings